server_maintain/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
@noahzaozao Can you review this module?
remove/update commented code
Add a README and update short headers/openerp/.